lib/core/test/unit/Cardano/Wallet/DB/Properties.hs

```diff
@@ -726,10 +726,13 @@ prop_rollbackTxHistory db@DBLayer{..} (InitialCheckpoint cp0) (GenTxHistory txs0
             unsafeRunExceptT $ initializeWallet wid cp0 meta mempty
             unsafeRunExceptT $ putTxHistory wid txs0
 
-    prop wid point = do
-        _ <- run $ unsafeRunExceptT $ mapExceptT atomically $ rollbackTo wid point
+    prop wid requestedPoint = do
+        point <- run $ unsafeRunExceptT $ mapExceptT atomically $ rollbackTo wid requestedPoint
         txs <- run $ atomically $ readTxHistory wid Descending wholeRange Nothing
-        monitor $ counterexample $ "\nTx history after rollback: \n" <> fmt txs
+
+        monitor $ counterexample $ "\n" <> "Actual Rollback Point:\n" <> (pretty point)
```

Comment:
This is weird. 
```
         Requested Rollback point:
         79.0

         Actual Rollback Point:
         0.0

         Original tx history:
         -  9eb44fb2 14.6 in ledger outgoing 2

         New tx history:
         -  9eb44fb2 14.6 in ledger outgoing 2
         Outgoing txs are reschuled ✗

  To rerun use: --match "/Cardano.Wallet.DB.MVar/MVar/rollback/Correctly re-construct tx history on rollbacks/"
```

Through experimentation it seems the `TxMetaSlot >. nearestPoint` predicate is ineffective. (If removed all txs are cleared). But still: wat?

Reply:
Not sure what fixed it 🤷‍♂️